pgstac_ingestor/schemas.py

```diff
@@ -0,0 +1,148 @@
+import base64
+import binascii
+import enum
+import json
+from datetime import datetime
+from typing import TYPE_CHECKING, Dict, List, Optional, Union
+from urllib.parse import urlparse
+
+from fastapi.encoders import jsonable_encoder
+from fastapi.exceptions import RequestValidationError
+from pydantic import (
+    BaseModel,
+    Json,
+    PositiveInt,
+    dataclasses,
+    error_wrappers,
+    validator,
+)
+from stac_pydantic import Collection, Item, shared
```

Comment:
I think we should drop the stac-pydantic requirement and use our own model.

stac-pydantic is unmaintained and it's not clear to me if it's going to be updated in the future